voluptuous/tests/tests.py

```diff
@@ -1528,3 +1528,19 @@ def test_any_with_discriminant():
         assert_equal(str(e), 'expected bool for dictionary value @ data[\'implementation\'][\'c-value\']')
     else:
         assert False, "Did not raise correct Invalid"
+
+def test_maybe_returns_subvalidator_error():
```

Comment:
Can you move that up to where the other Maybe test are so that they grouped together?

Reply:
I will merge as-is and then update the grouping via a separate change.